README.rst

```diff
@@ -34,7 +34,7 @@ Usage
       import numpy as np
       import mrinufft
       from mrinufft.trajectories import display
-      from mrinufft.trajectories.density import voronoi
+      from mrinufft.density import voronoi
```

Comment:
Do we need the voronoi as we know its like terribly slow? We can have it just there, but if no one is using it saves on maintainence

Reply:
It's not slow in 2D, which is why I used it in the examples.